examples/inspec-linux-audit/habitat/default.toml

```diff
@@ -51,6 +51,13 @@ token = '<automate_token>'
 user = '<automate_user>'
 environment = 'foo'
 
+# This is where you would put a waiver to skip a control
+[waivers]
+[waivers.control_id]
+expiration_date = "2019-10-23"
+run = false
+justification = "Blah Blah Blah"
+
```

Comment:
Do you have to have the ID of the control the waiver is applied to here?

Reply:
I see how it works now. Sorry, I did not see the below example.

Reply:
I think so this where I was struggling cause the yaml file needs to look like this
```yaml
control_id:
  run: false
  justification: Some justification
```

using the habitat templating I wasn't sure if there was a better like 
```toml
[waivers]
control_id = control id
  run = false
  justification = "Some justification"

another_control = another_control
  run = false
  justification = "Some justification"
```

I think the above though isn't proper toml.

Reply:
I am going to play around with the latest version of InSpec to understand waivers more.